### ISSUE TITLE
Error: Unsupported argument after '+'

### ISSUE THREAD

**Issue by PinyaColada** [andresgongora/synth-shell-greeter#36]
**Describe the bug**
When I open the terminal it prints Error: Unsupported argument after '+' and then it prints my configuration with shell-greeter completely normal

**To Reproduce**
I just installed and it started like this.

**Comment by olmari** [comment]
What OS? And(or what shell?

**Comment by pendenaor** [comment]
Having the same issue on WSL2 + Distrod + Archlinux system with `GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)`
```
Error: Unsupported argument after '+'
                     .                      OS              Arch Linux
                     #                      Kernel          5.10.60.1-microsoft-standard-WSL2
 ```

**Comment by andresgongora** [comment]
Hmmm I'm unable to replicate it. Do you 'bc' installed?

**Comment by pendenaor** [comment]
like that?
```
> bc --version
bc 1.07.1
```

**Comment by PinyaColada** [comment]
Hi, I have arch linux with bash and with kernel 6.0.2

**Comment by pendenaor** [comment]
find the culprit :smile:
in **synth-shell-greeter.sh**, function `getExternalIPv4` (line 1269)
```
$ drill +time=1 +tries=1 TXT -4 +short o-o.myaddr.l.google.com @ns1.google.com
Error: Unsupported argument after '+'
```

**Comment by andresgongora** [comment]
Awesome! I think it's fixed now as of 845e47a. Another solution, while this makes it upstream, is to install `dig` from the `dnsutils` package
